static/app/components/modals/widgetViewerModal.tsx

```diff
@@ -246,10 +249,7 @@ function WidgetViewerModal(props: Props) {
   // Get legends toggle settings from location
   // We use the legend query params for just the initial state
   const [disabledLegends, setDisabledLegends] = useState<{[key: string]: boolean}>(
-    decodeList(location.query[WidgetViewerQueryField.LEGEND]).reduce((acc, legend) => {
-      acc[legend] = false;
-      return acc;
-    }, {})
+    dashboardLegendUtils.getLegendUnselected(widget)
```

Comment:
Since `dashboardLegendUtils` keeps all this stuff in the URL now, is it possible to remove `disabledLegends` and `setDisabledLegends`?